src/scss/themes/_neugreysun.scss

```diff
@@ -0,0 +1,23 @@
+@use "../utils/neutheme";
+
+$bgColor: rgb(41, 41, 41);
+$textColor: #fff;
+$lightColor: #333;
+$shadowColor: #000;
+$activeColor: rgb(255, 217, 0);
+$disactiveColor: #777;
+
+@include neutheme.create-neu-theme(
+  "dark",
```

Comment:
This name must match the theme in `Editor.ts` file without the `neu-` prefix, please run a `yarn start` to try it